server/controllers/changeUsedFor/details.ts

```diff
@@ -0,0 +1,86 @@
+import FormWizard from 'hmpo-form-wizard'
+import { NextFunction, Response } from 'express'
+import { isEqual, sortBy } from 'lodash'
+import FormInitialStep from '../base/formInitialStep'
+import backUrl from '../../utils/backUrl'
+
+export default class ChangeUsedForDetails extends FormInitialStep {
+  middlewareSetup() {
+    this.use(this.setOptions)
+    super.middlewareSetup()
+  }
+
+  async setOptions(req: FormWizard.Request, res: Response, next: NextFunction) {
+    const token = await req.services.authService.getSystemClientToken(res.locals.user.username)
+    const usedForTypes = await req.services.locationsService.getUsedForTypes(token)
+    req.form.options.fields.usedFor.items = Object.entries(usedForTypes).map(([key, description]) => ({
+      value: key,
+      text: description,
+    }))
+    next()
+  }
+
+  locals(req: FormWizard.Request, res: Response): Record<string, any> {
+    const locals = super.locals(req, res)
+    const { location } = res.locals
+    const { id: locationId, prisonId } = location
+
+    const fields = { ...locals.fields }
+
+    if (!req.form.values?.usedFor) {
+      if (location.usedFor.length === 1) {
+        fields.usedFor.items = fields.usedFor.items.map((item: FormWizard.Field) => ({
+          ...item,
+          checked: location.usedFor.includes(item.value),
+        }))
+      }
+    }
```

Comment:
Instead of this, you should be able to override `getInitialValues`, like so:

```
  getInitialValues(req: FormWizard.Request, res: Response) {
    return {
      usedFor: res.locals.location.usedFor,
    }
  }
  ```

Reply:
@Thource are you happy with the approach @ashleygyngell has taken here?

Reply:
@Mjwillis Ah we discussed this yesterday and @Thource was happy with this.

Spoke with UCD team, there is a requirement for only pre-populating a checked checkbox when there is only one used for value.